Apps/CocoaPods-FCM/src/View/DashboardView.swift

```diff
@@ -27,30 +27,36 @@ struct DashboardView: View {
 
     @EnvironmentObject var userManager: UserManager
     var body: some View {
-        ZStack {
-            VStack {
-                SettingsButton {
-                    subscreenShown = .settings
+        ScrollView {
```

Comment:
First change is moving ScrollView to the top-most level. This fixes a bug I found where the settings button did nothing when you clicked on it on the Dashboard screen. After this change, the screen UI behaves as expected including the settings button is clickable.